resolwe_bio/tests/workflows/test_mirna.py

```diff
@@ -0,0 +1,29 @@
+from resolwe_bio.utils.test import BioProcessTestCase
+from resolwe.test import tag_process
+from resolwe.flow.models import Data
+
+
+class MicroRNATestCase(BioProcessTestCase):
+    @tag_process('workflow-mirna')
+    def test_mirna_workflow(self):
+        # Prepare data for aligning the reads with bowtie2 and annotation file for featureCounts.
+        with self.preparation_stage():
+            genome = self.prepare_genome('genome_rsem.fa.gz')
+            single_reads = self.prepare_reads(['reads rsem.fq.gz'])
+            annotation = self.prepare_annotation('annotation_rsem.gtf.gz')
+
+        inputs = {
+            'genome': genome.pk,
+            'reads': single_reads.pk,
+            'annotation': annotation.pk,
+            'feature_type': 'exon',
```

Comment:
The default value in the workflow inputs is `miRNA`. Why is there a difference?

Reply:
There is no GTF with miRNA. My suggestion would be wait with this until live test to see if we need a specific miRNA featureCounts test.